configure.ac

```diff
@@ -5654,6 +5654,15 @@ then
         AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_ALT_CERT_CHAINS"
     fi
 
+    if test "x$ENABLED_SESSION_TICKET" = "xno"
+    then
+        ENABLED_SESSION_TICKET="yes"
+        AM_CFLAGS="$AM_CFLAGS -DHAVE_TLS_EXTENSIONS -DHAVE_SESSION_TICKET"
+    fi
+
+    # FTPS server requires peristent pointer to session cache
```

Comment:
Let's just update this comment. The FTPS server requires a `NO_SESSION_CACHE_REF` to always have a session pointer. It does not need to be "peristent".